tools/bin/mtest

```diff
@@ -502,19 +507,23 @@ def get_crypto_config_file():
 
 def set_config_option(action, option, crypto_config_path, silent=True):
     force = [ "--force" ] if action == "set" else []
```

Comment:
Would be more readable if it was written like:

~~~~~
force = [] 
    if action == "set":
        force.append("--force")
        if "=" in option:
~~~~~